Ginger/GingerCoreCommon/Actions/ActOutDataSourceConfig.cs

```diff
@@ -48,42 +49,98 @@ public  static partial class Fields
 
         [IsSerializedForLocalRepository]
         public string DSTable { get; set; }
-        
-        [IsSerializedForLocalRepository]
-        public bool Active { get { return mActive; } set { mActive = value; OnPropertyChanged(Fields.Active); } }
 
-        [IsSerializedForLocalRepository]
-        public string OutputType { get; set; }
+        //[IsSerializedForLocalRepository]
+        //public bool Active { get { return mActive; } set { mActive = value; OnPropertyChanged(Fields.Active); } }
+
+        //[IsSerializedForLocalRepository]
+        //public string OutputType { get; set; }
 
         [IsSerializedForLocalRepository]
         public string OutParamMap { get; set; }
-        
+
         public override string ItemName
         {
             get
             {
-                return this.OutputType;
+                return this.DSName;
             }
             set
             {
-                this.OutputType = value;
+                this.DSName = value;
             }
         }
 
-        ObservableList<string> mPossibleValues = new ObservableList<string>();
-        public ObservableList<string> PossibleValues
+        //ObservableList<string> mPossibleValues = new ObservableList<string>();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4815237)